controllers/installation_controller.go

```diff
@@ -1071,17 +745,19 @@ func (r *InstallationReconciler) StartUpgrade(ctx context.Context, in *v1beta1.I
 		// node and are served by the local-artifact-mirror binary listening on localhost
 		// port 50000. we just need to get autopilot to fetch the k0s binary from there.
 		k0surl = "http://127.0.0.1:50000/bin/k0s-upgrade"
-		command, err := r.CreateAirgapPlanCommand(ctx, in)
+		command, err := artifacts.CreateAutopilotAirgapPlanCommand(ctx, r.Client, in)
```

Comment:
looks like this plan will also be executed twice in a typical upgrade workflow. any reason to keep this here? we removed the LAM jobs creation, we should probably remove this as well.

Reply:
removed